arcade/draw_commands.py

```diff
@@ -10,6 +10,11 @@
 import pyglet.gl as GL
 import pyglet.gl.glu as GLU
 
+from sympy import Symbol, nsolve
+import sympy
+import mpmath
+mpmath.mp.dps = 15
```

Comment:
Any way to do this without adding two new libraries? If the math can be done without sympy, then the users won't have to download and install. All for supporting one draw command they might not even use.